rust/tests/integration/direct_transport.rs

```diff
@@ -444,13 +444,17 @@ fn send_with_subchannels_succeeds() {
 
         // Must be received by dataConsumer1 and dataConsumer2.
         direct_data_producer
-            .send(WebRtcMessage::String(both.to_string()), None, None)
+            .send(
+                WebRtcMessage::String(std::borrow::Cow::Borrowed(both.as_bytes())),
```

Comment:
What about importing `std::borrow::Cow` since it is used so many times here?

Reply:
It was already imported, my fault. I've simplified to `Cow::Borrowed` everywhere.